Doc/library/stdtypes.rst

```diff
@@ -5456,6 +5463,165 @@ types, where they are relevant.  Some of these are not reported by the
       [<class 'bool'>]
 
 
+.. _int_max_str_digits:
+
+Integer string conversion length limitation
+===========================================
+
+CPython has a global limit for converting between :class:`int` and :class:`str`
+to mitigate denial of service attacks. This limit *only* applies to decimal or
+other non-power-of-two number bases. Hexidecimal, octal, and binary conversions
```

Comment:
spelling: `hexadecimal`

Reply:
good catch, thanks. fixed in all branches.